### PR TITLE
Feat/refactor e2e selector strategy

### PR DESCRIPTION
**Pull request by christopherferreira9**
- Adds a new logic for selectors to allow easier use of the page objects.
- Moved and made use of new constants

### CODE REVIEW

File: e2e/src/Utils.ts [hunk 1]
```diff
@@ -8,43 +8,30 @@ import FixtureServer from '../test/fixtures/FixtureServer';
...
-    if (driver.isIOS && locator.iosLocator !== undefined) {
-      return `${platformLocator.strategy}${locator.iosLocator.locator}`;
-    } else if (driver.isAndroid && locator.androidLocator !== undefined) {
-      // Explicitly check for driver.isAndroid in case we want to add web tests
-      return `${platformLocator.strategy}${locator.androidLocator.locator}`;
-    }
-    throw new Error('Platform locator is undefined');
+export const getSelectorForPlatform = (locator: MetaMaskElementSelector) => {
+  const platformSelector =
+    PLATFORM === 'IOS' ? locator.iosSelector : locator.androidSelector;
```
Comment (andreahaku) [inline e2e/src/Utils.ts:18]:
```
import { FIXTURE_SERVER_PORT, PLATFORM, Platforms } from './Constants';
...
PLATFORM === Platforms.IOS
```

**Review by andreahaku — COMMENTED** [review]
one small change/fix
The rest is ok

File: e2e/src/Utils.ts (hunk 1 above)
Comment (christopherferreira9) [reply e2e/src/Utils.ts]: done!

**Review by andreahaku — APPROVED** [review]
LGTM

**Review by omridan159 — APPROVED** [review]

**Pull request merged**